lib/location/location_core.c

```diff
@@ -267,11 +267,6 @@ int location_core_validate_params(const struct location_config *config)
 		return -EINVAL;
 	}
 
-	if ((config->interval > 0) && (config->interval < 10)) {
```

Comment:
This change as such is good and we should keep it. It just started me thinking if somebody configures GNSS as only method and interval == 0, should we change the method_gnss implementation so that GNSS runs in continuous mode without being start/stopped constantly?

Reply:
What was the use case for this interval == 0? Probably this is used only for testing, so I'm not sure we should add implementation for that. In the beginning we did decide that we don't support continuous tracking. 😃

Reply:
`interval == 0` means that it's a single position query. Description of the configuration says:
`Set to 0 for a single position update`
If customers ask for GNSS continuous mode, we should probably add separate configuration flags for it.

Reply:
Okay, I thought interval == 0 means continuous. And yes, we decided not to support continuous GNSS tracking. So let's stick with that for the time being at least.

Reply:
Of course, I somehow thought that the intention was to enable interval == 0, but it's actually 1-9 second intervals.